src/components/AudioPlayer.vue

```diff
@@ -1,87 +1,122 @@
 <template>
   <div>
-    <v-bottom-sheet
-      transition="slide-y-transition"
+    <v-footer
       v-if="playerActive"
       v-model="playerActive"
-      class="audio-player"
       dark
-      height="100"
-      hide-overlay
-      persistent
+      height="playerHeight"
+      fixed
+      padless
+      style="z-index: 10000;"
     >
       <v-card
+        class="audio-player-card"
         tile
         width="100%"
         :height="playerHeight"
```

Comment:
Here it's `:height="playerHeight"` while in `<v-footer>` above it's `height="playerHeight"`.

Perhaps we want to be consistent here?